### DISCUSSION TITLE
JobArrayTaskLimit

### DISCUSSION THREAD

**Discussion by xingzhe1129** [nextpyp/.github#32] [Q&A] [answered]
Hi,
I am running nextPYP particle refinement jobs on our cluster and it worked very well. I requested 6 nodes and 5 nodes are running and one node is pending with the reason of JobArrayTaskLimit, which I guess that each job is limited to 5 nodes. And I see there are about 20 nodes on our server. I asked the administer of our cluster and he said that the cluster didn't have 5 nodes limitation for each job. So we guess it is the limitation of the software. Do you meet this before and do you know the node limitation? I am running the data set of 10304, which there are 12 tilt series, and there are 12 nodes available on our cluster and I want to run 12 jobs parallelly. Thanks.

**Comment by abartesaghi** [comment, marked as answer]
nextPYP should not be limiting the resources unless you set `Max number of threads` in the **Resources** tab (advanced options) to a value >0. What value are you using for this parameter? If >0, setting it back to 0 should remove the JobArrayTaskLimit.

**Reply by xingzhe1129** [reply]
Yes, Just now I see the default  Max number of threads under advanced options is 500, which makes sense, and I used 5 nodes, each node has 100 threads. Thank you so much for your quick response.
